.github/workflows/docker-image-scan.yml

```diff
@@ -0,0 +1,35 @@
+name: Platform's docker images vulnerability scan
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+
+jobs:
+  scan-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run trivy vulnerability scanner for the OpenHIM core image
```

Comment:
Are we meant to be checking the core and console images here if they already have their own action notifying us? I assume this action should only do scanning for the platform image itself?